taxi_service/settings.py

```diff
@@ -124,6 +124,10 @@
 
 STATIC_URL = "static/"
 
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
```

Comment:
Why do you use path join in one place but just "/" in another?

Reply:
I did / according to default way, provided in lower part of file which was by default, but when seen "path"  in templates was thinking about cross-platform. good point btw.